app/src/main/java/org/gdg/frisbee/android/eventseries/EventListFragment.java

```diff
@@ -189,47 +202,4 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         ButterKnife.inject(this, v);
         return v;
     }
-
-    @Override
-    public void onClick(View view) {

```

Comment:
Yeah, I was going to mention this but I forgot. Sorry. 

If we tracked the share button, can someone look at the numbers? I personally don't think that is used too much? 

In most of the apps like Google+, it requires 2 click. 

Or we can add overflow menu to the all items in the list. Thats an option.